ledger/metrics/src/main/scala/com/daml/metrics/Metrics.scala

```diff
@@ -708,6 +708,17 @@ final class Metrics(val registry: MetricRegistry) {
 
       // Meters how long processing of a request takes
       val httpRequestTimer: Timer = registry.timer(Prefix :+ "http_request_timing")
+      // Meters how long processing of a command submission request takes
+      val commandSubmissionTimer: Timer = registry.timer(Prefix :+ "command_submission_timing")
+      // Meters how long processing of a query request takes
+      val queryTimer: Timer = registry.timer(Prefix :+ "query_timing")
+      // Meters how long processing of a party management request takes
+      val partyManagementTimer: Timer = registry.timer(Prefix :+ "party_management_timing")
+      // Meters how long processing of a package management request takes
+      val packageManagementTimer: Timer = registry.timer(Prefix :+ "package_management_timing")
+      // Meters how long parsing and decoding of incoming json payload takes
+      val incomingJsonParsingAndValidationTimer: Timer =
+        registry.timer(Prefix :+ "incoming_json_parsing_and_validation_timing")
```

Comment:
It's probably fine to have all command endpoints tracked by a single timer, but shouldn't we rather have more fine-grained metrics to allow users to differentiate between the time taken to, say, make a query or fetch request?

Reply:
I can do that, I'm just unsure about what is most useful. Maybe I should just go the route "More metrics = better" :smile:

Reply:
Not necessarily, I think that grouping commands together in sensible, but I would at least differentiate for things that clearly achieve different things, like between uploading and downloading a package.

Reply:
did split them up with 33842a5